media/css/firefox/releasenotes.scss

```diff
@@ -417,3 +417,51 @@ $image-path: '/media/protocol/img';
         }
     }
 }
+
+.release-note {
+    .release-note-progressive-rollout-indicator {
+        @include text-body-md;
+        text-align: center;
+        width: 230px;
+        border: 2px solid $color-marketing-gray-20;
+        border-radius: $border-radius-md;
+        padding: $spacing-sm 0;
+        margin: auto;
+
+        img {
+            width: 100px
+        }
+
+        p {
+            margin-bottom: $spacing-sm;
+        }
+
+        details {
+            @include text-body-xs;
+            cursor: help;
```

Comment:
Suggest changing this to `pointer`
```suggestion
            cursor: pointer;
```

Reply:
Actually, I would only apply this to `<summary>`, since right now it applies to the entire content, and not just the part that toggles to expand/collapse.